ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -178,3 +179,19 @@ def get_element_type(element_name):
     """
 
     return (f"//a[contains(@title,'{element_name}')]", By.XPATH)
+
+
+def get_blockpool_ui_element(pool_name, OCP_Version):
```

Comment:
I didn't understand this function

Reply:
The function returns the UI element of the Blackpool, which is better for keeping the HTML script in a separate file. as it was requested by Oded here: https://github.com/red-hat-storage/ocs-ci/pull/6169#discussion_r955897571